spec/support/sign_in_helper.rb

```diff
@@ -0,0 +1,19 @@
+module SignInHelper
+  def sign_in
+    password = "shared_password"
+    create(:configuration, name: "admin:password",
+           value: BCrypt::Password.create(password))
+    visit root_path
+    fill_login_form(password: password)
+  end
+
+  def fill_login_form(password:)
+    fill_in "login", with: "test@example.com"
+    fill_in "password", with: password
+    click_on "Let me in!"
+  end
+end
+
+RSpec.configure do |c|
+  c.include SignInHelper, type: :feature
```

Comment:
Agreed. And besides [ControllerMacros](https://github.com/dradis/dradis-ce/blob/dix-global-search/spec/support/controller_macros.rb) already provides this functionality.